mobile-ui/src/main/res/values/navigation_action_bar_styles.xml

```diff
@@ -65,8 +65,8 @@
         <item name="isSkipHidden">false</item>
         <item name="isShadowHidden">false</item>
         <item name="isSkipAllowed">true</item>
-        <item name="primaryActionColor">@color/butterscotch500</item>
-        <item name="primaryActionTitleColor">@color/appTextDark</item>
+        <item name="primaryActionColor">@color/colorSecondary</item>
+        <item name="primaryActionTitleColor">@color/colorTextSecondary</item>
```

Comment:
@sierrawang where is `colorTextSecondary` defined? Is there a file that needs to be added to the commit?